packages/sync/src/modules/class-users.php

```diff
@@ -49,7 +49,18 @@ public function name() {
 	 * @return string
 	 */
 	public function table_name() {
-		return 'users';
+		return 'usermeta';
```

Comment:
Is this expected?

Reply:
we calculate the `WHERE`  against columns in this table.